src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs

```diff
@@ -2996,6 +3070,19 @@ PropertyDescriptorCollection ICustomTypeDescriptor.GetProperties(Attribute[]? at
             }
         }
 
+        private sealed class ResetEventContext
+        {
+            public ResetEventContext(ManualResetEvent resetEvent, int ownerId)
+            {
+                ResetEvent = resetEvent;
+                OwnerId = ownerId;
+            }
+
+            public ManualResetEvent ResetEvent { get; }
+
+            public int OwnerId { get; }
```

Comment:
```suggestion
            public readonly ManualResetEvent ResetEvent { get; }

            public readonly int OwnerId { get; }
```

Reply:
Wait a minute. Currently `ResetEvent` and `OwnerId` is readonly property (they have only `get` accessor). You mean, that I should change readonly property to readonly field?

Reply:
Sorry; no change needed